### PR TITLE
Fixed flaky test

### PR DESCRIPTION
**Pull request by Astashevich**

### CODE REVIEW

File: src/main/java/by/onliner/page/components/ShoppingCartPopup.java [hunk 1]
```diff
@@ -24,7 +26,12 @@ public void waitForComponentOpened() {
      * @return visibility of popup
      */
     public boolean isShoppingCartPopupVisible() {
-        return shoppingCartPopupCloseButton.isDisplayed();
+       try {
+           Waiter.waitForVisibility(shoppingCartPopupCloseButton, 2, 500);
+           return true;
```
Comment (Anassstasi) [inline src/main/java/by/onliner/page/components/ShoppingCartPopup.java]: Why do you return true not shoppingCartPopupCloseButton.isVisible() ?

Comment (Astashevich) [reply src/main/java/by/onliner/page/components/ShoppingCartPopup.java]: Fixed

File: src/main/java/by/onliner/utils/Waiter.java [hunk 2]
```diff
@@ -22,4 +25,14 @@ public static void elementToBeClickable(WebElement element) {
     public static void waitForVisibility(WebElement element) {
         wait.until(ExpectedConditions.visibilityOf(element));
     }
+
+    /***
+     * Waits until the element is displayed using custom timing
+     */
+    public static void waitForVisibility(WebElement element, int timeoutSec, int pollingMils) {
```
Comment (Anassstasi) [inline src/main/java/by/onliner/utils/Waiter.java]: Create a method where you pass only timeout value as an argument and polling is default. The chance that you will need custom polling value is very law.

Comment (Astashevich) [reply src/main/java/by/onliner/utils/Waiter.java]: Ok

File: src/main/java/by/onliner/page/components/ShoppingCartPopup.java [hunk 3]
```diff
@@ -24,7 +26,12 @@ public void waitForComponentOpened() {
      * @return visibility of popup
      */
     public boolean isShoppingCartPopupVisible() {
-        return shoppingCartPopupCloseButton.isDisplayed();
+       try {
+           Waiter.waitForVisibility(shoppingCartPopupCloseButton, 1, 100);
```
Comment (Anassstasi) [inline src/main/java/by/onliner/page/components/ShoppingCartPopup.java]: Remove polling from here.

Comment (Astashevich) [reply src/main/java/by/onliner/page/components/ShoppingCartPopup.java]: Removed

**Review by Anassstasi — APPROVED** [review]

**Pull request merged**